### ISSUE TITLE
Feature Request: Player:ChangeName() & Player:ChangeAdjective()

### ISSUE THREAD

**Issue by pineappledan** [LoneGazebo/Community-Patch-DLL#6205]
I am wondering if a command could be added to change a player's name and adjective? 

I am working on a custom Phoenician civ for VP that can found new city-states. It pulls cities from the unused player list normally used for rebelling cities to make new CSs. The cities retain their name and pull from Phoenicia's city list correctly, but even if they didn't there is a City:ChangeName() command.

It would really add to the polish for this civ if Beirut, Baalbek, etc. actually stayed as Beirut and Baalbek in the city-state diplomacy screen, rather than Geneva, Malacca, etc, and if their units all said "Phoenician Warrior" instead of "Prague Warrior". The relevant columns are Civilizations.ShortDescription and Civilizations.Adjective. Maybe for the sake of completion another command could be made to change Civilizations.Description (The "XXX'ian Empire" text). I imagine someone could make good use of that for a civ that evolves through the ages. Carolingians ==> Merovingian ==>French empire, etc.

TL;DR - I want a way to change Civilizations.ShortDescription and Civilizations.Adjective while in-game.

**Comment by JFDaly** [comment]
You can currently do this with PreGame.SetCivilizationAdjective(playerID, string) and PreGame.SetLeaderName(playerID, string) (CivilizationShortDescription and Description also exist). The caveat is that these overwrite any nicknames the player has given themselves, but that's only applicable for MP games.

**Comment by pineappledan** [comment]
So like this? 
local pMinorName = pCity:GetName()
PreGame.SetCivilizationAdjective(player, "Phoenician")
PreGame.SetCivilizationShortDescription(player, pMinorName)
PreGame.SetCivilizationDescription(player, pMinorName)

**Comment by JFDaly** [comment]
You need to pass the player's ID (player:GetID() if you don't already have it) not the player object, but otherwise yeah, that'll work

**Comment by pineappledan** [comment]
Right. I'm using "for player in pairs(Players) do" earlier, so player is the ID, but that wouldn't have been clear
Thank you for the help!

**Comment by pineappledan** [comment]
experimented with this, but minor civs can't be subject to SetCivilizationAdjective, SetCivilizationShortDescription, or SetCivilizationDescription.

I am specifically interested in changing the name and adjective of city-states
